include/wx/osx/cocoa/dataview.h

```diff
@@ -501,7 +501,7 @@ class wxCocoaDataViewControl : public wxWidgetCocoaImpl,
     virtual wxDataViewColumn* GetColumn(unsigned int pos) const wxOVERRIDE;
     virtual int GetColumnPosition(wxDataViewColumn const* columnPtr) const wxOVERRIDE;
     virtual bool InsertColumn(unsigned int pos, wxDataViewColumn* columnPtr) wxOVERRIDE;
-    virtual void FitColumnWidthToContent(unsigned int pos) wxOVERRIDE;
+    virtual void FitColumnWidthToContent(unsigned int pos, bool fitRowHeight = false) wxOVERRIDE;
```

Comment:
As usual, I think the code would be much more readable if we used an enum rather than bool for this parameter, i.e. `int fitFlags = 0` with the possibility to pass `Fit_RowHeight`.

Although looking at the code this really could be called `Force_AllRows` too because it seems that all passing `true` here does is to set `calculateAllRows` to `true` in the code.

Reply:
> Although looking at the code this really could be called Force_AllRows too

`Fit_RowHeight` is semantic and says "we need to update height calculation when called from this place". That doing that forces iteration over all rows is an implementation detail that may change.

OTOH, it only is used in one place (Thaw) where "force all calculations" *is* appropriate meaning.